### DISCUSSION TITLE
Revised Memo Thread

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#25] [Assignments]
### Due 4/29

Feel free to post your revised memo here when you would like me to review it.

**Comment by penelopelamb** [comment]
https://docs.google.com/document/d/1SDURCaoLo6TZAHLAzLEsBFkwoq1QmT54mR6YNdFi1fw/edit?usp=sharing

**Comment by willa-21** [comment]
https://docs.google.com/document/d/1oNlmaNkFBhE87tA2xJ77YnEgUnRHy29786SlHvXmvQw/edit?usp=sharing

**Comment by codingdianat** [comment]
Steven, Diana https://docs.google.com/document/d/1ntyzKAuOQ5EyIGzjUt9FMpkmcq12B52i-Wkx9gCwVVY/edit?usp=sharing

**Comment by kericsp** [comment]
Kristen, Emma, Keri https://docs.google.com/document/d/1luEGoedbzhedIfqYMA7nWkhUxx3iNjDuX2HC2Sblv8w/edit?usp=sharing

**Comment by joonyloony** [comment]
https://docs.google.com/document/d/1F7eX4HvRmtT9miAq7_OGafAOFyV_5kV4Jy7CEx_vau4/edit?usp=sharing

**Comment by lindseymardona** [comment]
https://docs.google.com/document/d/1rTA8Ie0XUcuqYRIij91iTfYFv1I48S7zOmdjDv41UCs/edit#

**Comment by NishuKrish** [comment]
https://docs.google.com/document/d/1kZpRNve7cEyXLhCrZJw2k9Qs9fWhWY3x6NW1lwuPXxQ/edit?usp=sharing

Nish and Ivy

**Comment by kaittli** [comment]
https://docs.google.com/document/d/11kF8Yhpas4TA_AIQmPYa3XQkxAKFmcE_4WMvCwtOn9E/edit?usp=sharing
Antony and Kaitlyn
